test_common/harness/errorHelpers.cpp

```diff
@@ -105,6 +105,10 @@ const char *IGetErrorString(int clErrorCode)
         case CL_INVALID_SPEC_ID: return "CL_INVALID_SPEC_ID";
         case CL_MAX_SIZE_RESTRICTION_EXCEEDED:
             return "CL_MAX_SIZE_RESTRICTION_EXCEEDED";
+        case CL_INCOMPATIBLE_COMMAND_QUEUE_KHR:
```

Comment:
Do we also want `CL_INVALID_COMMAND_BUFFER_KHR`?

Reply:
Yep, might as well.